src/java.base/share/classes/javax/crypto/spec/IvParameterSpec.java

```diff
@@ -47,30 +47,30 @@ public class IvParameterSpec implements AlgorithmParameterSpec {
      *
      * @param iv the buffer with the IV. The contents of the
      * buffer are copied to protect against subsequent modification.
-     * @throws NullPointerException if <code>iv</code> is <code>null</code>
+     * @throws NullPointerException if {@code iv} is {@code null}
      */
     public IvParameterSpec(byte[] iv) {
         this(iv, 0, iv.length);
     }
 
     /**
-     * Creates an IvParameterSpec object using the first <code>len</code>
-     * bytes in <code>iv</code>, beginning at <code>offset</code>
+     * Creates an IvParameterSpec object using the first {@code len}
+     * bytes in {@code iv}, beginning at {@code offset}
      * inclusive, as the IV.
      *
      * <p> The bytes that constitute the IV are those between
-     * <code>iv[offset]</code> and <code>iv[offset+len-1]</code> inclusive.
+     * {@code iv[offset]} and {@code iv[offset+len-1]} inclusive.
      *
-     * @param iv the buffer with the IV. The first <code>len</code>
-     * bytes of the buffer beginning at <code>offset</code> inclusive
+     * @param iv the buffer with the IV. The first {@code len}
+     * bytes of the buffer beginning at {@code offset} inclusive
      * are copied to protect against subsequent modification.
-     * @param offset the offset in <code>iv</code> where the IV
+     * @param offset the offset in {@code iv} where the IV
      * starts.
      * @param len the number of IV bytes.
-     * @throws IllegalArgumentException if <code>iv</code> is <code>null</code>
+     * @throws IllegalArgumentException if {@code iv} is {@code null}
      * or {@code (iv.length - offset < len)}
-     * @throws ArrayIndexOutOfBoundsException is thrown if <code>offset</code>
-     * or <code>len</code> index bytes outside the <code>iv</code>.
+     * @throws ArrayIndexOutOfBoundsException if {@code offset}
+     * or {@code len} index bytes outside the {@code iv}.
```

Comment:
`RC2ParameterSpec(int, byte[], int)` ctor should be documented with AIOOBE as well. If you pass a negative offset into it, AIOOBE is thrown by the `System.arraycopy()` call.

Reply:
fixed